generators/connection/index.js

```diff
@@ -37,7 +37,7 @@ module.exports = class ConnectionGenerator extends Generator {
       mariadb: 'mysql',
       mysql: 'mysql',
```

Comment:
Could you also change this to `mysql2` while we're at it? That's what I got a complaint about.